api/src/main/java/io/opentelemetry/trace/Span.java

```diff
@@ -22,6 +23,31 @@
 @ThreadSafe
 public interface Span extends ImplicitContextKeyed {
 
+  /**
+   * Returns the {@link Span} from the current {@link Context}, falling back to a default, no-op
+   * {@link Span} if there is no span in the current context.
+   */
+  static Span current() {
```

Comment:
Question from the SIG meeting today: would it make sense to have a method on `ImplicitContextKeyed` for retrieving the item from the Context as well? Seems like having both saving and retrieval would be a win.

Reply:
It came to mind, but these need to be static so can't :)

Reply:
hmmm. Yeah, no static methods on an interface. This is what happens when I try to reason about things the morning after the 10pm meeting the night before. ;)